Automattic/wp-calypso
# Add/free flow create site domain generation

#### Proposed Changes

* Pass site title (blog title) to createSite function for free flow step creation step.

#### Testing Instructions

1. Check out this PR
2. Setup backend diff: D96193-code
3. Go through the free flow (`/setup/free`).



#### Pre-merge Checklist





Related to #
